kratos/utilities/time_discretization.h

```diff
@@ -0,0 +1,112 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Philipp Bucher
+//
+
+#if !defined(KRATOS_TIME_DISCRETIZATION_H_INCLUDED )
+#define  KRATOS_TIME_DISCRETIZATION_H_INCLUDED
+
+// System includes
+#include <array>
+
+// External includes
+
+// Project includes
+
+namespace Kratos {
+namespace TimeDiscretization {
+
+class KRATOS_API(KRATOS_CORE) BDF1
+{
+public:
+    std::array<double, 2> ComputeBDFCoefficients(double DeltaTime) const;
+};
+
+class KRATOS_API(KRATOS_CORE) BDF2
+{
+public:
+    std::array<double, 3> ComputeBDFCoefficients(double DeltaTime) const;
+
+    std::array<double, 3> ComputeBDFCoefficients(double DeltaTime,
+                                                 double PreviousDeltaTime) const;
+};
+
+class KRATOS_API(KRATOS_CORE) BDF3
+{
+public:
+    std::array<double, 4> ComputeBDFCoefficients(double DeltaTime) const;
+};
+
+class KRATOS_API(KRATOS_CORE) BDF4
+{
+public:
+    std::array<double, 5> ComputeBDFCoefficients(double DeltaTime) const;
+};
+
+class KRATOS_API(KRATOS_CORE) BDF5
+{
+public:
+    std::array<double, 6> ComputeBDFCoefficients(double DeltaTime) const;
+};
+
+class KRATOS_API(KRATOS_CORE) BDF6
+{
+public:
+    std::array<double, 7> ComputeBDFCoefficients(double DeltaTime) const;
+};
+
+class Newmark
+{
+public:
+    double GetBeta()  const { return 0.25; }
```

Comment:
@RiccardoRossi you want those to be input too I assume?
Then I would make those values the defaults in the constructor

Reply:
ok for making the. defaults

Reply:
done